WhatsApp/app/src/main/res/layout/fragment_notifications.xml

```diff
@@ -1,6 +1,564 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/root_layout"
```

Comment:
dont forget to follow the naming convention. Should be svNotifications or something alike